web/apps/labelstudio/src/components/Menubar/Menubar.jsx

```diff
@@ -132,7 +132,12 @@ export const Menubar = ({ enabled, defaultOpened, defaultPinned, children, onSid
         <div className={menubarClass}>
           <Dropdown.Trigger dropdown={menuDropdownRef} closeOnClickOutside={!sidebarPinned}>
             <div className={`${menubarClass.elem("trigger")} main-menu-trigger`}>
-              <img src={absoluteURL("/static/icons/logo.svg")} alt="Label Studio Logo" height="22" />
+              <img
+                src={absoluteURL("/static/icons/logo.svg")}
+                alt="Label Studio Logo"
+                height="22"
```

Comment:
if we're adding inline style for height lets remove the hight attribute. although ideally we should a css class here and define the height that way for white labeling purposes
```suggestion
```